GOVERNANCE.md

```diff
@@ -0,0 +1,55 @@
+# Governance
+
+This document describes the governance process under which the Serverless
+Working Group (WG) will manage this repository.
+
+## Working Group Meetings
+
+In order to provide equitable rights to all Working Group members,
+the following process will be followed:
+
+* Official WG meetings will be announced at least a week in advance.
+* Proposed changes to any document will be done via a Pull Request (PR).
+* PRs will be reviewed during official WG meetings.
+* During meetings, priority will be given to PRs that appear to be ready for
+  a vote over those that appear to require discussions.
+* PRs should not be merged if they have had substantial changes made within
+  two days of the meeting.
+  Rebases, typo fixes, etc. do not count as substantial.
+  Note, administrivia PRs that do not materially modify WG output documents
+  may be processed by WG admins as needed.
+* Resolving PRs ("merging" or "closing with no action") will be done as a
+  result of a motion made during a WG meeting, and approved.
+* Reopening a PR can be done if new information is made available, and a
+  motion to do so is approved.
+* Any motion that does not have "unanimous consent" will result in a formal
+  vote. See [Voting](#voting).
+
+## PRs
```

Comment:
Do we need to raise an issue that the PR addresses first?

Reply:
I know some groups like to do that, but I think its kind of silly overhead to require an issue. I'm personally ok with someone opening a PR and explaining why they think the change is needed directly in the PR. I think having it all in one spot anyway  ;-)